internal/sidekick/dart/templates/lib/main.dart.mustache

```diff
@@ -25,6 +25,14 @@ limitations under the License.
 {{/Codec.DocLines}}
 library;
 
+// ignore_for_file: argument_type_not_assignable {{! dynamic is used for protobuf messages }}
+// ignore_for_file: avoid_unused_constructor_parameters {{! `fromJson` may not use its arguments if the message is empty }}
+// ignore_for_file: camel_case_types {{! Nested messages have the parent/child separated by an underscore }}
+// ignore_for_file: comment_references {{! TODO(#25): improve the generated dartdocs }}
+// ignore_for_file: implementation_imports {{! imports of protobuf and rpc implementations }}
+// ignore_for_file: lines_longer_than_80_chars {{! TODO(#25): improve the generated dartdocs }}
```

Comment:
This one would be fine to ignore in the long-term in my opinion. Some of the lints don't matter much for generated code and I think this one qualifies.

Reply:
I think that one will get fixed as part of correctly parsing the proto docs.